<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWizGame</class>
 <widget class="QWizard" name="CWizGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lgck-builder.qrc">
    <normaloff>:/images/lgckbuilder64ronnie2.png</normaloff>:/images/lgckbuilder64ronnie2.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::IndependentPages</set>
  </property>
  <widget class="QWizardPage" name="pageIntro">
   <widget class="QLabel" name="sWelcome">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to the Game Wizard</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>This wizard will guide you the process of creating a new game.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="lgck-builder.qrc">:/images/bitmap1.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="pageGeneral">
   <property name="subTitle">
    <string>General</string>
   </property>
   <widget class="QLineEdit" name="eName">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>61</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>66</y>
      <width>53</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please provide general information about your new game.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>53</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eAuthor">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>104</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="eDescription">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Game description</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="pageSpeed">
   <property name="subTitle">
    <string>Game Speed</string>
   </property>
   <widget class="QGroupBox" name="gDifficulty">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Difficulties</string>
    </property>
    <widget class="QComboBox" name="cbSpeedNormal">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>56</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nightmare</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSpeedNightmare">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSpeedHell">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>86</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hell</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>115</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Insane</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSpeedInsane">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>109</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>26</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="rSetByDifficulty">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>17</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed set by difficulty</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rSetByLevel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed set by level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Default speed</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDefaultSpeed">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>224</y>
      <width>78</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;NOTE: Game speed can be set by level or by difficulty. &lt;/p&gt;&lt;p&gt;If chose the later, the per level speed setting will be ignored in favor of the difficulty presets.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="pageFeatures">
   <property name="subTitle">
    <string>Features</string>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Default Sounds</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gAutoCenter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Auto center</string>
    </property>
    <widget class="QComboBox" name="cbACSpeed">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>26</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbACGranular">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>56</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Granularity</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="cLookup">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow Free Lookup</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>146</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Lives&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbHPMax">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>78</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbLives">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>78</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>186</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Hit Points&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>115</y>
      <width>53</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Max&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbHP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>78</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbTimeout">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>78</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>226</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Timeout&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>115</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Initial setting&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbLivesMax">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>78</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="pageFinal">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="lgck-builder.qrc">:/images/bitmap1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Congratulation, you have just created a new game. Click to the finish button to complete the process.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rSetByDifficulty</tabstop>
  <tabstop>cbSpeedNormal</tabstop>
  <tabstop>cbSpeedNightmare</tabstop>
  <tabstop>cbSpeedHell</tabstop>
  <tabstop>cbSpeedInsane</tabstop>
  <tabstop>rSetByLevel</tabstop>
  <tabstop>eName</tabstop>
  <tabstop>eAuthor</tabstop>
  <tabstop>eDescription</tabstop>
 </tabstops>
 <resources>
  <include location="lgck-builder.qrc"/>
 </resources>
 <connections/>
</ui>
